.circleci/config.yml

```diff
@@ -33,31 +33,32 @@ jobs:
       - run:
           name: Build source
           command: node common/scripts/install-run-rush.js rebuild --verbose
-      - persist_to_workspace:
-          root: .
+      - save_cache:
+          key: build-cache-{{ .Environment.CIRCLE_WORKFLOW_ID }}
           paths:
-            - '*/dist'
-            - '*/lib'
-            - '*/node_modules'
-            - common/temp
-            - gen-server
-            - gen-client
+            - ~/.rush
+            - .
```

Comment:
We may not need to persist `.git` to cache if it becomes too heavy.

Reply:
Agree, at least we can remove the checkout step in the dependant jobs.